### DISCUSSION TITLE
Total new be 'in trouble"

### DISCUSSION THREAD

**Discussion by wmrmclaren** [community/community#23902] [New to GitHub]
<p>hi all<br>
i am new to the git hub community, and mainly here because i use homebridge on a PI for my apple homekit<br>
the thing is that i found a plug in for my denon smart speaker but cant seem te get it to work (not even sure if its meant for my model etc etc)<br>
i would like to get in to contact with the author of the code<br>
i cant seem to find a way to send a message, is this not possible or am i just blind ?</p>
<p>love to hear form you all<br>
with regards<br>
mitch</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="wmrmclaren" data-post="1" data-topic="198279">
<div class="title">
<div class="quote-controls"></div>
 wmrmclaren:</div>
<blockquote>
<p>i would like to get in to contact with the author of the code<br>
i cant seem to find a way to send a message, is this not possible or am i just blind ?</p>
</blockquote>
</aside>
<p>There is no “private message” functionality on GitHub. If there’s a problem with the code or documentation creating an issue in the repository is usually the right step. Make sure to read the README (if any) first, though. Good luck! :slightly_smiling_face:</p>
